msgs/confirmSigMsg.go

```diff
@@ -0,0 +1,52 @@
+package msgs
+
+import (
+	"github.com/FourthState/plasma-mvp-sidechain/plasma"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+const (
+	// ConfirmSigMsgRoute is used for routing this message.
+	ConfirmSigMsgRoute = "confirmSig"
+)
+
+// SpendMsg implements the RLP interface through `Transaction`
+type ConfirmSigMsg struct {
+	Input1    plasma.Input
```

Comment:
do we have to restrict to 2 inputs. Why not include as many inputs as someone likes. []Input